lib/money-heuristics/heuristics.rb

```diff
@@ -10,10 +10,10 @@ module Heuristics
     # currency in an entire sentence
     #
     # Returns: Array (matched results)
-    def analyze(str)
+    def analyze(str, *filters)
```

Comment:
I'd suggest using keyword arguments here, this will allow us to keep the method backwards compatible in case we'll need to add more options